internal/gps/vcs_version_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gps
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/golang/dep/internal/test"
+)
+
+func TestVCSVersion(t *testing.T) {
+	test.NeedsExternalNetwork(t)
+	test.NeedsGit(t)
+
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	h.TempDir("src")
+	gopath := h.Path(".")
+	h.Setenv("GOPATH", gopath)
+
+	importPaths := map[string]struct {
+		rev      Version
+		checkout bool
+	}{
+		"github.com/pkg/errors": {
+			rev:      NewVersion("v0.8.0").Pair("645ef00459ed84a119197bfb8d8205042c6df63d"), // semver
+			checkout: true,
+		},
+		"github.com/Sirupsen/logrus": {
```

Comment:
nit: with all the case sensitivity issues this is causing, let's go with the new one - sirupsen